Description: If init script uses files from /usr, $remote_fs is needed as Dependency
Author: Marc Haber <mh+debian-packages@zugschlus.de>
Forwarded: via web form
Last-Update: 2016-08-07
--- a/atop.init
+++ b/atop.init
@@ -7,8 +7,8 @@
 #
 ### BEGIN INIT INFO
 # Provides:             atop
-# Required-Start:	$local_fs
-# Required-Stop:	$local_fs
+# Required-Start:	$local_fs $remote_fs
+# Required-Stop:	$local_fs $remote_fs
 # Default-Start:	2 3 4 5
 # Default-Stop:		0 1 6
 # Short-Description: Advanced system and process activity monitor
--- a/atopacct.init
+++ b/atopacct.init
@@ -7,8 +7,8 @@
 #
 ### BEGIN INIT INFO
 # Provides:             atopacct
-# Required-Start:	$local_fs
-# Required-Stop:	$local_fs
+# Required-Start:	$local_fs $remote_fs
+# Required-Stop:	$local_fs $remote_fs
 # Default-Start:	2 3 4 5
 # Default-Stop:		0 1 6
 # Description:          This daemon switches on process accounting and
